cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -586,6 +590,11 @@ def run_call_back(self, status_kw_name='action') -> typing.Tuple[str, typing.Uni
                                             email_status='email sent')
             else:
                 job.write_dataserver_status(status_dictionary_value=status, full_dict=self.par_dic)
+        except MultipleDoneEmail as e:
```

Comment:
this should not, however, break the current situation when the product is eventually recieved by the user, after several "submit-done" cycles. So write_dataserver_status should still happen, perhaps with a note that it's duplicated done.